### PR TITLE
feat: generate hbase ranger policy #101

### LINKED ISSUE #101: HBase Ranger policies missing for tdp_user
Currently the HBase Ranger policies are missing for tdp_user. This makes `test-cluster-deployment` fail.
The solution is to create it by hand in ranger, but it could be implemented through ansible, the same way hive and hdfs policies are created.

### PR DESCRIPTION
**Pull request by gonzaloetjo**
#### Which issue(s) this PR fixes:

Fixes #101 

#### Additional comments:
- Add creation of tdp_user hbase policies.
- Correct 'test-cluster-deployment' to provide errors. 

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: deploy-users.yml [hunk 1]
```diff
@@ -39,6 +39,16 @@
       with_items:
         - "{{ user_hive_policies }}"
 
+- name: Create Hbase access policies in Ranger
```
Comment (leopaul36) [inline deploy-users.yml:42]: Should be "HBase"

File: inventory/group_vars/all.yml [hunk 2]
```diff
@@ -213,3 +218,31 @@ user_hive_policies:
...
+        values: ["testTable"]
+    policyItems:
+      - users: [tdp_user]
+        groups: []
+        delegateAdmin: true
+        accesses:
+          - { isAllowed: true, type: Read }
+          - { isAllowed: true, type: Write }
+          - { isAllowed: true, type: Create }
+        conditions: []
```
Comment (leopaul36) [inline inventory/group_vars/all.yml:248]: No new line at end of file

File: test-cluster-deployment.yml [hunk 3]
```diff
@@ -169,3 +199,9 @@
     failed_when: _result.rc != 0
     tags:
       - hbase
+
+  - name: Log previous action stdout to console
+    debug: 
+      msg: '{{ _result.stdout }}'
+    tags:
+      - hbase
```
Comment (leopaul36) [inline test-cluster-deployment.yml:207]: No new line at end of file

**Review by leopaul36 — CHANGES_REQUESTED** [review]
Couple changes as stated by the comments otherwise all good !

**Review by Nuttymoon — APPROVED** [review]

**Pull request merged**